frontend/common/SliderServerClient.js

```diff
@@ -39,6 +40,23 @@ export const nothing_actions = ({ actions }) =>
         ])
     )
 
+const sliders_error = (e, previously_set) => {
+    const el = previously_set ? document.querySelector(`bond[def=${previously_set.values().next().value}]`) : null
+    if (e.status === 503) {
+        open_pluto_popup({
+            type: "warn",
+            body: "Sliders are still starting, check back later for interactivity!",
+            source_element: el,
+        })
+    } else {
+        open_pluto_popup({
+            type: "warn",
```

Comment:
I'm thinking to possibly change this from `warn` to `error`

Reply:
I think it already looks a bit too scary... I see the sliderserver functionality more as a bonus that people get when reading the article, and when PSS fails it just means that there is no bonus today, not that there is something to get worried about. Also because it is not something that the user could fix.